packages/components/pie-textarea/src/textarea.scss

```diff
@@ -49,16 +50,28 @@
       box-shadow: none;
       outline: none;
     }
+
+    &::placeholder {
+      color: var(--textarea-placeholder-color);
```

Comment:
Worth noting (not sure if there's a ticket for this, but probably worth one) – placeholder colour changed in the latest changelog as the current colour is too dark (check Design Changelog v1.8.0)

Reply:
We'll capture this in a ticket that's coming imminently